Add information about ethereum nodes and how they are used (#4572)
rotki/rotki: docs/usage_guide.rst

This setting modifies the rounding mechanism choosing from ``Round up``, ``Round down`` and ``Half even`` (rounds to the nearest value). It can be individually modified how amounts and values are rounded.

Ethereum RPC nodes
^^^^^^^^^^^^^^^^^^^^^

This setting allows you to modify the nodes that will be queried to interact with the Ethereum blockchain. We provide a list of public nodes but sometimes they are not available or are overloaded, this is why is a good idea to have more nodes just in case.
When making queries we always give preference to your own nodes if you have any and then add a small list of randomly selected open nodes in case they are needed. If you don't have your own node then the open nodes are always selected. The weight of the node sets the probability

> of picking it and is represented by a percentage. From that list is also possible to mark them as active or inactive by switching on or off the toggle button.

Reviewer: ```suggestion
of picking it and is represented by a percentage. From that list is also possible to mark them as active or inactive by using the toggle button.
```
```suggestion
of picking it and is represented by a percentage. From that list is also possible to mark them as active or inactive by switching on or off the toggle button.
```

> .. image:: images/rotki_nodes_managment.png

Reviewer: filename `managment` -> `management`

   :alt: Customizing the app's connection to ethereum nodes
   :align: center

In this menu you can also edit, delete or add more nodes.

.. image:: images/rotki_nodes_managment_edition.png
   :alt: Editing an ethereum node
   :align: center

> The slider lets you modify the weight of the node and you can also manually imput a value. The owned parameter gives priority over the rest of not owned nodes to the one that you are editing and you can also mark the node as not active in this screen to completely ignore it in the selection process.

Reviewer: ```suggestion
The slider lets you modify the weight of the node and you can also manually imput a value. The owned parameter gives priority to the selected node over the rest of not owned nodes. You can also mark the node as not active in this screen to completely ignore it in the selection process.
```
```suggestion
The slider lets you modify the weight of the node and you can also manually imput a value. The owned parameter gives priority over the rest of not owned nodes to the one that you are editing and you can also mark the node as not active in this screen to completely ignore it in the selection process.
```

> Similarly you can add your own node.

Reviewer: ```suggestion
Similarly you can add more nodes.
```

This behavior can be disabled by turning this setting off.

> Omit ETH staking events

Reviewer: use api description
